src/components/Table/Table.tsx

```diff
@@ -18,7 +21,7 @@ const b = block('table');
 
 export default class Table extends React.Component<TableProps & ClassNameProps> {
     render() {
-        const {content, legend, marker = 'disk', className, caption} = this.props;
+        const {content, legend, hideLegend, marker = 'disk', className, caption} = this.props;
```

Comment:
Maybe we can check if legend exists? In what case we can pass a legend and hide it?

Reply:
Now we have this part of code https://github.com/gravity-ui/page-constructor/blob/main/src/components/Table/Table.tsx#L44-L48

And if change logic here, it will break all created tables